README.md

```diff
@@ -127,52 +136,59 @@ For more information, click [here](https://docs.aws.amazon.com/cognito/latest/de
 
-### Testing binary resources
+### Testing Binary resources
 
 **Using Postman** 
 
-To test, use Postman.  For steps, click [here](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/README.md#using-postman-to-make-api-requests).
+For steps, click [here](https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/README.md#using-postman-to-make-api-requests).
 
-**Note**: We recommend you to test binary resources by using the `Binary` folder in Postman. 
+**Note**: We recommend you test Binary resources by using the `Binary` folder in Postman. 
 
 **Using cURL**
 
 To test this with cURL, follow these steps:
 1.	POST a binary resource to FHIR API using the following command:
++ For Windows:
+```sh
+curl -H "Accept: application/json" -H "Authorization:<COGNITO_AUTH_TOKEN>" -H "x-api-key:<API_KEY>" --request POST ^
+  --data '{"resourceType": "Binary", "contentType": "image/jpeg"}' ^
+  <API_URL>/Binary
+```
++ For Mac/Linux:
 ```sh
 curl -H "Accept: application/json" -H "Authorization:<COGNITO_AUTH_TOKEN>" -H "x-api-key:<API_KEY>" --request POST \
   --data '{"resourceType": "Binary", "contentType": "image/jpeg"}' \
   <API_URL>/Binary
 ```
-2. Check the POST's response. There will be a presignedPutUrl parameter. Use that pre-signed url to upload your file. See below for command
+2. Check the POST's response. There will be a `presignedPutUrl` parameter. Use that pre-signed url to upload the file, using this command:
 ```sh
 curl -v -T "<LOCATION_OF_FILE_TO_UPLOAD>" "<PRESIGNED_PUT_URL>"
 ```
 
 ### Testing bulk data export
 
-Bulk Export allows you to export all of your data from DDB to S3. We currently support the [System Level](https://hl7.org/fhir/uv/bulkdata/export/index.html#endpoint---system-level-export) export. For more information about bulk export, refer to the FHIR [Implementation Guide](https://hl7.org/fhir/uv/bulkdata/export/index.html).
+Bulk Export allows you to export all of the data stored in DynamoDB to S3. We currently support the [System Level](https://hl7.org/fhir/uv/bulkdata/export/index.html#endpoint---system-level-export) export. For more information about bulk export, refer to the relevant FHIR [Implementation Guide](https://hl7.org/fhir/uv/bulkdata/export/index.html).
```

Comment:
Suggested: For more information about bulk export, see FHIR [Implementation Guide].